src/tests/ftest/util/server_utils_params.py

```diff
@@ -333,6 +333,9 @@ class PerEngineYamlParameters(YamlParameters):
                 "CRT_SWIM_RPC_TIMEOUT=10"],
             "ofi+verbs": [
                 "FI_OFI_RXM_USE_SRX=1"],
+            "ofi+cxi": [
+                "FI_OFI_RXM_USE_SRX=1",
```

Comment:
the FI_OFI_RXM_USE_SRX is not needed for cxi;  only needed when the provider is <>;ofi_rxm